docs/build/smart-contracts/deploy/private-network.md

```diff
@@ -0,0 +1,364 @@
+```
+
+```js
+[+] Using env: {
+  DOCKER_RPC_HTTP_URL: 'http://host.docker.internal:8545/',
+  DOCKER_LISTEN: '0.0.0.0:4000',
+  DOCKER_DISABLE_TRACER: 'false',
+  DOCKER_DEBUG: '0'
+}
+[+] Open in the browser: http://localhost:4000
+ Network blockscout_default  Creating
+ Network blockscout_default  Created
+ Container blockscout-db-1  Creating
+ Container blockscout-frontend-1  Creating
+ Container blockscout-smart-contract-verifier-1  Creating
+ Container blockscout-redis_db-1  Creating
+ Container blockscout-smart-contract-verifier-1  Created
+ Container blockscout-db-1  Created
+ Container blockscout-frontend-1  Created
+ Container blockscout-redis_db-1  Created
+ Container blockscout-backend-1  Creating
+ Container blockscout-backend-1  Created
+ Container blockscout-frontend-1  Starting
+ Container blockscout-redis_db-1  Starting
+ Container blockscout-smart-contract-verifier-1  Starting
+ Container blockscout-db-1  Starting
+ Container blockscout-db-1  Started
+ Container blockscout-redis_db-1  Started
+ Container blockscout-smart-contract-verifier-1  Started
+ Container blockscout-backend-1  Starting
+ Container blockscout-frontend-1  Started
+ Container blockscout-backend-1  Started
+```
+
+**Step 2:** To access this block explorer, open up [http://localhost:4000](http://localhost:4000) in your browser. 
+
+
+Step 3: Copy and paste the deployed contract address in the search field and press Enter. You should see the recently deployed contract.
+
+![](/img/build/smart-contracts/pn-verify-tx-block-explorer.png)
+
+## Interacting with deployed contract <a id="interacting-with-deployed-contract"></a>
+
+### using hardhat utils contract task
+
+1. To call a read-only function of the deployed contract, run the command below:
+
+```js
+hh call KaiaGreeter getTotalGreeting
+```
+
+![](/img/build/smart-contracts/pn-read-function.png)
+
+
+2. To send a function invoking transaction to the deployed contract, run the command below:
+
+```js
+hh send KaiaGreeter greet
+```
+
+```jsx title="Result"
+sent KaiaGreeter#greet (tx: 0xc0bd25ffb594c13d5ae1f77f7eb02f2978013c69f9f6e22694b76fa26c329e85)...ok (block 2837, gas used: 47457)
+```
+
+### using Kaia SDK
+
+**Step 1:** To interact with the deployed contract using [Kaia SDK](https://github.com/kaiachain/kaia-sdk), you need to install Kaia SDK by running this command:
+
+```js
+npm install --save @kaiachain/ethers-ext
+```
+
+**Step 2:** In the Explorer pane, Create a new folder called "utils" and click the New File button to create a new file named `kaia-sdk.js` in the utils folder.
+
+Step 3:  Copy and paste the following code inside the file.
+
+```js
+const { JsonRpcProvider, Wallet } = require("@kaiachain/ethers-ext");
+const { ethers } = require("ethers");
+const provider = new JsonRpcProvider("http://127.0.0.1:8545/")
+ 
+const privKey = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" // copy any private key provided by the local private network
```

Comment:
same here private keys and public keys need not be given. but Hightlight to user about this as Environment variables.